### ISSUE TITLE
Capturing with Windows 11 (ARM) on M2 platforms

### ISSUE THREAD

**Issue by tim-rex** [baldurk/renderdoc#2949]
## Description

I'm filing this as a feature request as I suspect this is probably not currently a supported use case (thus, not strictly a bug/issue)..

I'm trying to use RenderDoc on M2 hardware running Windows 11 (ARM) in a VM (Parallels)

I've had some limited success with x86 and x64 binaries (supported on Windows 11 ARM via built-in emulation). The captures appear to work, RenderDoc warns about degraded support due to software emulation fallback.

<img width="527" alt="image" src="https://github.com/baldurk/renderdoc/assets/19639392/0b16ada2-54e0-4302-ac82-dc047dde23c1">

I'm unable to capture against ARM64 binaries. RenderDoc is unable to perform DLL injection, very probably because RenderDoc itself is an x64 application trying to inject to an arm64 binary.. It has every right to complain.

<img width="527" alt="image" src="https://github.com/baldurk/renderdoc/assets/19639392/6cad7ccf-93f5-4cd9-a393-d2a291e800c9">

Perhaps this would be solved with a native arm64 build of RenderDoc.. though I suspect it's not quite so straight forward as that.

To summarise, these are the scenarios:
- Capture an x86 application running on Windows 11 (ARM) on Apple M2 hardware  (OpenGL capture works via software mode)
- Capture an x64 application running on Windows 11 (ARM) on Apple M2 hardware  (OpenGL capture works via software mode)
- Capture an ARM64 application running on Windows 11 (ARM) on Apple M2 hardware

## Environment

* RenderDoc version: 1.27
* Operating System: Windows 11 (ARM) running under Parallels 18 on an M2 Macbook Pro
* Graphics API: OpenGL

**Comment by baldurk** [comment]
Indeed as you suspected this is not a supported use case, ARM is only supported on Android and it is not supported on any other platforms. I have no plans to add support for it.
